app/models/like.rb

```diff
@@ -0,0 +1,4 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post, counter_cache: :likes_counter
+end
```

Comment:
- [ ] Like model should have a method that updates the likes counter for a post so would you add this method as well?